### DISCUSSION TITLE
Did something change? My faceid results are different post update

### DISCUSSION THREAD

**Discussion by Soulreaver90** [Mikubill/sd-webui-controlnet#2567] [General]
I had updated this morning and everything broke. I saw things were fixed so I updated and ip adapter (faceid specifically) is working again. The issue I am noticing is that none of my previous seeds are reproducible, I am getting completely different results! Now the results aren't bad, but none of my previous seeds are reproducible. I forced cloned and restored my previously used commit (7660993), and everything is fine again. Did something change recently in the logic that would effect results? Can someone else replicate?

**Comment by huchenlei** [comment]
Working on the fix rn.

**Reply by huchenlei** [reply]
Fixed in #2569.
